Resources/LogBackends/MessageQueueBackend.py

```diff
@@ -32,6 +35,14 @@ def __init__(self, backendParams=None):
     """
     Initialization of the MessageQueueBackend
     """
+    # The `Format` parameter is passed as `fmt` to libJsonFormatter
+    # which uses it to know which "standard" fields to keep in the
+    # json output. So we need these
+    if not backendParams:
+      backendParams = {'Format': DEFAULT_FMT}
+    elif 'Format' not in backendParams:
+      backendParams['Format'] = DEFAULT_FMT
```

Comment:
Rather that duplicating, how about:
```suggestion
    if backendParams is None:
      backendParams = {}
    if 'Format' not in backendParams:
      backendParams['Format'] = DEFAULT_FMT
```
or more concisely:
```suggestion
    if backendParams is None:
      backendParams = {}
    backendParams.setdefault("Format", DEFAULT_FMT)
```

Reply:
Good idea